### ISSUE TITLE
Supplemental Figure 1: Benchmarking of Alevin-fry vs. Cell Ranger

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-paper-figures#4]
We should mention some of the benchmarking that we did to compare results between libraries processed with Cell Ranger and Alevin-fry to help explain why we built our workflow with Alevin-fry. This probably does not need to be a main figure and can be a supplemental figure (unless others have different opinions). 

This figure should include: 

- [ ] Comparison of run time and memory usage between both methods 
- [ ] Comparison of total UMI/cell 
- [ ] Comparison of total genes/cell
- [ ] Comparison of mean gene expression 

All of the plots except for the run time/ memory usage exist as plots in the [FAQ: Why did we use Alevin-fry for processing?](https://scpca.readthedocs.io/en/stable/faq.html#why-did-we-use-alevin-fry-for-processing). We should polish those figures and move the code for generating those figures to this repo.

**Comment by allyhawkins** [comment]
So I started working on this, in particular, I was going back through the previous time and memory comparison we used to compare Cell Ranger and Alevin-fry runs. The original benchmarking results can be found in https://github.com/AlexsLemonade/alsf-scpca/blob/main/analysis/quantifier-comparisons/03-memory_time_comparison.Rmd. 

I have a folder of trace files that were generated during benchmarking that were used in that comparison that I can use, but there are a few caveats. 

- We used Cell Ranger v6, which does not retain introns for single-cell samples. This did come out after we did our original benchmarking, but do we want to re-run with Cell Ranger v7 for better comparisons? I don't think this is imperative, but I did want to note the difference in the Cell Ranger versions. Note that in the images we have in the FAQ's, we use Cell Ranger v6. 
- For Alevin-fry we used a reference without intronic reads for single-cell samples in that particular analysis. But we did other later benchmarking runs with the index we use now, which we used to generate the FAQs. The problem is that I don't have the trace files for those, so I can't get the time or memory information. I will, at minimum, need to re-generate the trace files for the Alevin-fry runs. 
- For all of these runs, we used Ensembl 103 for the indices. If we don't want to re-run Cell Ranger, then I will need to use that older index for the re-runs of Alevin-fry. But if we want to be more up-to-date with what's on the portal, we should use Ensembl 104, which would mean re-runs of the benchmarking runs for both Cell Ranger and Alevin-fry that we want to use to create the supplemental figure.

Here's what I think we should do here: 

- Re-run 6 samples (3 cell, 3 nuclei) through Cell Ranger v6 but with Ensembl 104. 
- Re-run the same 6 samples through Alevin-fry with the same running conditions we use on the portal (splici index, Ensembl 104) 

Doing this will create both new trace files to grab time and memory from as well as new results files to re-generate the UMI, genes detected, and mean gene expression comparisons. 

Note that we should probably update the FAQ figures to also include the data from the new re-runs. 

Tagging @jaclyn-taroni and @jashapiro for any thoughts on this approach.
